src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+  const sourceStringArr = sourceString.split('\n').join('').split(';')
+    .map(item => item.split(':')).filter(item => item.length > 1)
```

Comment:
the filter can be replaced by checking if elements exist before adding them to an object.